ci-operator/jobs/infra-image-mirroring.yaml

```diff
@@ -1381,3 +1381,40 @@ periodics:
     - configMap:
         name: image-mirror-mappings
       name: config
+- agent: kubernetes
+  cluster: app.ci
+  cron: '@yearly'
+  decorate: true
+  labels:
+    ci.openshift.io/area: image-mirroring-on-demand
+    ci.openshift.io/role: image-mirroring
+  name: periodic-image-mirroring-on-demand
+  spec:
+    containers:
+    - args:
+      - -c
+      - |
+        set -o errexit
+        failures=0
+        for mapping in $IMAGE_PATHS; do
+          image_name_tag=${mapping##*/}
+          cmd="oc image mirror --keep-manifest-list ${mapping} registry.ci.openshift.org/ci/${image_name_tag} --skip-multiple-scopes"
+          echo "Running: ${cmd}"
+          if ! ${cmd}; then
+            echo "ERROR: Failed to mirror images from $mapping"
+            failures=$((failures+1))
+          fi
+        done
+        exit $failures
+      command:
+      - /bin/bash
+      env:
+      - name: HOME
```

Comment:
Not sure if this is needed?

Reply:
ah, it is needed for the push credentials which are lacking in that deployment, @hongkailiu what is our procedure for creating push credentials?

Reply:
@hongkailiu @jmguzik I am missing something here.
We want to be able to mirror into openshift ci (same as done periodically in `core-services/image-mirroring/supplemental-ci-images/mapping_supplemental_ci_images_ci`).
Why do we need a new secret?

Reply:
@rnetser I did not know tbh that @hongkailiu documented this part in our docs. I suggest to follow it to make it working. You can still keep the customization and relay on ENV to provide secrets.